Release/bin/x64/plugins/cyber_engine_tweaks/mods/AppearanceMenuMod/Modules/props.lua

```diff
@@ -1514,10 +1530,13 @@ function Props:SavePreset(preset, path, fromDB)
       table.insert(lights, light)
     end
 
+    -- this will always increment the preset name O_O    
```

Comment:
Is this possibly required for versioning or something?

Reply:
I have no idea what the initial idea was, but if I'd change `MyPresetName` (previously in `MyPresetName.json`), the best case was that I'd end up with the preset `MyPresetName` in `MyPresetName (1).json`. Most of the time, I'd have a duplicate of the preset with the same name. Consecutive attempts of saving either would then create `MyPresetName (1)` in `MyPresetName (2).json` and so on.

Max was too swamped to even look at the problem, and this solution seems to be working so far. The only change I've noticed is that the selected preset doesn't get properly initialized; when I switch to the presets tab, it will always unload the current one to re-initialize the first list item. Maybe I'll do a second round of fixes for that one.